skimage/measure/_regionprops.py

```diff
@@ -631,7 +631,7 @@ def _parse_docs():
     import re
     import textwrap
 
-    doc = regionprops.__doc__
+    doc = regionprops.__doc__ or ''
```

Comment:
When calling _install_properties_docs on a null string, does it now behave as a no-op?

Reply:
effectively. the regex then just returns nothing.